packages/dds/tree/src/test/feature-libraries/modular-schema/genericFieldSnapshots.test.ts

```diff
@@ -57,4 +57,5 @@ export function testSnapshots() {
 
 const baseContext = {
 	originatorId: snapshotSessionId,
+	revision: undefined,
```

Comment:
Would be good to add a test where the revision is defined and the changeset's inner content also has that revision so that we can check the produced snapshot doesn't have the nested revision.

Reply:
Actually, scratch that, it's probably best to just add it to the revisionTagCodec.spec.ts.